server/app/views/admin/AdminLayout.java

```diff
@@ -137,16 +129,13 @@ private NavTag renderNavBar() {
         .with(
             headerLink(
                 "API keys", apiKeysLink, activeNavPage == NavPage.API_KEYS ? activeNavStyle : ""))
-        .with(headerLink("Logout", logoutLink, Styles.FLOAT_RIGHT));
+        .with(headerLink("Logout", logoutLink, "float-right"));
   }
 
   private ATag headerLink(String text, String href, String... styles) {
     return a(text)
         .withHref(href)
         .withClasses(
-            Styles.PX_3,
-            Styles.OPACITY_75,
-            StyleUtils.hover(Styles.OPACITY_100),
-            StyleUtils.joinStyles(styles));
+            "px-3", "opacity-75", StyleUtils.hover("opacity-100"), StyleUtils.joinStyles(styles));
```

Comment:
Did we decide to not inline modifiers like StyleUtils.hover?

Reply:
from what I remember, that wasn't explicitly mentioned, but maybe in-lining style prefixes could be a future PR that would further reduce the tailwind.css file

Reply:
Future PR sounds good.